src/main/java/hudson/tasks/junit/TestResult.java

```diff
@@ -934,4 +968,14 @@ public CaseResult getCase(String suiteName, String transformedFullDisplayName) {
                 .findFirst()
                 .orElse(null);
     }
+
+    private transient PrintStream logger;
+
```

Comment:
I definitely do not consider this as a really elegant way of logging :) but not sure what are the other options to get some debug/output in the console when running tests.

Reply:
https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/JenkinsRule.html#showAgentLogs-hudson.model.Slave-org.jvnet.hudson.test.LoggerRule- will probably do what you want.

Reply:
great I didn't know this option